reader.py

```
@@ -1,19 +1,21 @@
 import zipfile
-from PySide6.QtWidgets import QApplication, QWidget, QPushButton, QLabel, QVBoxLayout, QMainWindow, QHBoxLayout,QHBoxLayout, QSpacerItem, QSizePolicy
+from PySide6.QtWidgets import QApplication, QWidget, QPushButton, QLabel, QVBoxLayout, QMainWindow, QHBoxLayout,QHBoxLayout, QToolButton, QToolBar, QDialog
```

Flake8 (7.2.0)

[error] redefinition of unused 'QHBoxLayout' from line 2

(F811)